pkg/tsdb/cloudwatch/models/cloudwatch_query.go

```diff
@@ -63,7 +64,7 @@ type CloudWatchQuery struct {
 	AccountId         *string
 }
 
-func (q *CloudWatchQuery) GetGMDAPIMode(logger log.Logger) GMDApiMode {
+func (q *CloudWatchQuery) GetGMDAPIMode() GMDApiMode {
```

Comment:
A bit unrelated to this pr but can we document what gmd stands for here?

Reply:
I'm renaming this method to `GetGetMetricDataAPIMode`